docs/content/topics/scripting/guide.md

```diff
@@ -9,1401 +9,867 @@ aliases:
 ## A Basic Brigade Script
 
-Here is a basic `brigade.js` file that just sends a single
-message to the log:
+Here is a basic `brigade.js` script that just sends a single message to the
+log:
 
 ```javascript
-console.log("hello world")
+console.log("Hello, World!");
 ```
-[brigade-01.js](../../examples/brigade-01.js)
+[01-hello-world](https://github.com/brigadecore/brigade/tree/v2/examples/01-hello-world)
 
-If we run this script, we would see output that looked something like this:
+First let's create the example project:
 
 ```console
-Started brigade-worker-01brwzs64rve2jvky87hxy1wsp-master
-yarn start v0.27.5
-$ node prestart.js
-prestart: src/brigade.js written
-$ node --no-deprecation ./dist/src/index.js
-hello world
-Done in 1.44s.
+$ brig project create --file examples/01-hello-world/project.yaml
+
+Created project "hello-world".
 ```
 
-> Tip: You can use the `brig`  command to send brigade.js
-> files to Brigade.
->
-> In this tutorial we run scripts with `brig run --file brigade.js $PROJECT`, where `$PROJECT` is a
-> project ID like `brigade-830c16d4aaf6f5490937ad719afd8490a5bcbef064d397411043ac`.
+Next, let's trigger execution of the project script by creating an event:
 
-In essence, all we have done is started a brigade session, logged "hello world", and exited.
+```console
+$ brig event create --project hello-world --follow
 
-This example executes the log in the _global scope_. While it's fine to work in the
-global scope, most of the time what we want to do with Brigade is write _event handlers_.
+Created event "261229dc-1140-4f6a-bf91-bd2a69f31721".
+
+Waiting for event's worker to be RUNNING...
+2021-09-20T22:15:12.047Z INFO: brigade-worker version: a398ba8-dirty
+Hello, World!
+```
+
+This example executes the log in the _global scope_. While it's fine to work
+in the global scope, most of the time what we want to do with Brigade is write
+_event handlers_.
 
 ## Brigade Events and Event Handlers
 
-Brigade listens for certain things to happen, such as a new push on a GitHub repository or an
-image update on DockerHub. (The events that it listens for are configured in your
-project).
+Brigade listens for certain things to happen, such as a new push on a GitHub
+repository or an image update on DockerHub. (The events that it listens for are
+configured in your project under the `eventSubscriptions` section).
 
-When Brigade observes such an event, it will load the `brigade.js` file and see if there
-is an _event handler_ that matches the _event_.
+When Brigade observes such an event, it will load the project script and
+determine if an _event handler_ matching the _event_ exists.
 
 Using the `brig` tool introduced above, we can see how this works.
 
-The `brig` tool triggers an `exec` event (short for _execute_) on Brigade. So our
-`brigade.js` file can intercept this event and respond to it:
+The `brig` tool triggers an `exec` event (short for _execute_) on Brigade. So
+our script can intercept this event and respond to it:
```

Comment:
I understand why this is in this section and not the previous one-- because in the previous example, we unconditionally responding to all subscribed events the same, so the notion of an "event" first becomes relevant in this example.

That said, I feel like it broke the flow of the doc for me seeing `brig event create` explained _after_ having already used it.